config/estuary.go

```diff
@@ -34,27 +34,32 @@ func (cfg *Estuary) Save(filename string) error {
 	return save(cfg, filename)
 }
 
-// Sets the root of many paths
-func (cfg *Estuary) SetDataDir(ddir string) {
-	cfg.StagingDataDir = updateRootDir(ddir, cfg.DataDir, cfg.StagingDataDir)
-	cfg.NodeConfig.UpdateRoot(ddir, cfg.DataDir)
-	cfg.DataDir = ddir
-}
+func (cfg *Estuary) SetRequiredOptions() error {
+	//TODO validate required options values - check empty strings etc
 
-func NewEstuary() *Estuary {
+	cfg.StagingDataDir = filepath.Join(cfg.DataDir, "stagingdata")
+	cfg.ServerCacheDir = filepath.Join(cfg.DataDir, "cache")
+	cfg.NodeConfig.WalletDir = filepath.Join(cfg.DataDir, "estuary-wallet")
+	cfg.NodeConfig.DatastoreDir = filepath.Join(cfg.DataDir, "estuary-leveldb")
+	cfg.NodeConfig.Libp2pKeyFile = filepath.Join(cfg.DataDir, "estuary-peer.key")
 
-	pwd, _ := os.Getwd()
+	if cfg.NodeConfig.Blockstore == "" {
+		cfg.NodeConfig.Blockstore = filepath.Join(cfg.DataDir, "estuary-blocks")
+	}
+	return nil
+}
 
-	cfg := Estuary{
-		DataDir:                pwd,
-		StagingDataDir:         filepath.Join(pwd, "stagingdata"),
+func NewEstuary() *Estuary {
+	return &Estuary{
+		DataDir:                ".",
```

Comment:
I'm not sure that's what we want as default. If we're running estuary and our `pwd` is `/home/gabe` but my estuary folder and bin are at `/home/gabe/estuary`, I would want the default folder to be `/home/gabe/estuary`.

Reply:
This PR is to make changes to how things were, just before `pwd` was introduced. Also, `"."` works as expected when `ddir` is not specified